Mathlib/Algebra/Polynomial/Degree/Definitions.lean

```diff
@@ -1215,6 +1215,9 @@ theorem eq_one_of_monic_natDegree_zero (hf : p.Monic) (hfd : p.natDegree = 0) :
   rw [Monic.def, leadingCoeff, hfd] at hf
   rw [eq_C_of_natDegree_eq_zero hfd, hf, map_one]
 
+theorem Monic.natDegree_eq_zero (hf : p.Monic) : p.natDegree = 0 ↔ p = 1 :=
```

Comment:
```suggestion
theorem Monic.natDegree_eq_zero_iff (hf : p.Monic) : p.natDegree = 0 ↔ p = 1 :=
```

Reply:
Actually here I think the `_iff` is pretty clearly implied by the rest